agent/internal/config/jwt.go

```diff
@@ -0,0 +1,36 @@
+package config
+
+import (
+	"fmt"
+	"github.com/form3tech-oss/jwt-go"
```

Comment:
It was not this package that I've suggested, if you check the repository this is archived and read-only. Use the one mentioned in the comment.

Reply:
I've used the same package which is already being used in your codebase there `agent/internal/grpc/grpc.go`

The link you provided does not work, it gives 400 status code

I'm assuming you meant that one package? https://pkg.go.dev/github.com/golang-jwt/jwt
I will use it, it seems to be alive and have a lot of users

Reply:
do you want legacy jwt library to be remained in the codebase and me - introducing a new library only in the value setter & validator? or may I refactor the part with the old jwt library as well?

Reply:
I've made the validation with the new library, but have not touched the parts in the existing codebase.

Also I've slightly changed the tests because the behaviour of the library is different from what I've had before.

Reply:
It would be nice if you could replace it everywhere.